src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -71,6 +71,9 @@ public function setup_view( $arguments ) {
 		$this->set_view( $view );
 
 		$this->enqueue_assets( $context, $view );
+
+		// Ensure widgets never get Filter Bar classes on their containers
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 236c761d639b37ab5bbca9da6cbb7eaf</sub>

Reply:
```suggestion
		// Ensure widgets never get Filter Bar classes on their containers.
```